### ISSUE TITLE
Some font colors in the dark theme are not readable 

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-python#12750]
![IMG_1298](https://github.com/user-attachments/assets/32ff0aa8-eb56-4ef0-a5f8-7789eb54b5a5)

https://mne.tools/stable/auto_tutorials/preprocessing/35_artifact_correction_regression.html#references

**Comment by scott-huberty** [comment]
Also, as seen in the [Eyetracking tutorial:](https://mne.tools/stable/auto_tutorials/preprocessing/90_eyetracking_data.html)

<img width="782" alt="Screenshot 2024-08-05 at 9 06 49 AM" src="https://github.com/user-attachments/assets/49273be6-7338-456e-8663-7ce0330573da">

**Comment by drammock** [comment]
working on this right now

**Comment by hoechenberger** [comment]
> Also, as seen in the [Eyetracking tutorial:](https://mne.tools/stable/auto_tutorials/preprocessing/90_eyetracking_data.html)

Ha, I think this was one of those "we'll fix it later" bits back when we merged my revised HTML reprs. I'll take full blame for this one; however I won't have time to look into this before September or so. But happy to test on macOS if @drammock can come up with a solution before that
